guacamole-docker/bin/start.sh

```diff
@@ -404,6 +404,37 @@ END
     ln -s /opt/guacamole/radius/guacamole-auth-*.jar "$GUACAMOLE_EXT"
 }
 
+## Adds properties to guacamole.properties which select the OPENID
+## authentication provider, and configure it to connect to the specified OPENID
+## provider.
+##
+associate_openid() {
+
+    # Verify required parameters are present
+    if [ -z "$OPENID_AUTHORIZATION_ENDPOINT" -o -z "$OPENID_CLIENT_ID" ]; then
```

Comment:
According to the OpenID documentation (http://guacamole.apache.org/doc/gug/openid-auth.html), the following parameters are required:
- openid-authorization-endpoint
- openid-jwks-endpoint
- openid-issuer
- openid-client-id
- openid-redirect-uri

Seems like we should probably check for all of these as required, here, and not just the authorization endpoint and client ID?

Reply:
makes sense indeed, pushed the changes